projects/storefrontlib/cms-components/content/paragraph/paragraph.module.ts

```diff
@@ -17,8 +29,19 @@ import { ParagraphComponent } from './paragraph.component';
         },
       },
     }),
+    { provide: COMPILER_OPTIONS, useValue: {}, multi: true },
+    {
+      provide: CompilerFactory,
```

Comment:
We will not add this JitCompiler. This will increase the bundle size. For the link in paragraph component, can we extract the link, then create an angular link for it?

Reply:
I've found that angular needs to compile a new component in order to process a template with angular properties. Utilizing the router in our case.

This is the best solution I can find for imitating the router behaviour without dynamic compilation: https://stackoverflow.com/a/62783788/12566149